client/src/route/auth/AuthProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { AuthProvider as OIDCAuthProvider } from 'react-oidc-context';
+import { useOidcConfig } from '../../util/auth/useOidcConfig';
+
+interface AuthProviderProps {
+  children: React.ReactNode;
+}
+
+const AuthProvider: React.FC<AuthProviderProps> = ({ children }) => {
+  const oidcConfig = useOidcConfig();
+
+  if (!oidcConfig) {
+    return <div>Loading...</div>;
```

Comment:
a different message was suggested for this in an earlier PR.

Reply:
![image](https://github.com/INTO-CPS-Association/DTaaS/assets/78600714/63e9b35d-3698-4ff7-b28e-57a23a54afff)

Reply:
It seems `<div>Loading...</div>` is an indication of authentication provider not being available. If so, you can say `<div>Authentication service unavailable...try again later</div>`